lib/Onnxifi/GlowOnnxifiManager.cpp

```diff
@@ -27,12 +25,17 @@ GlowOnnxifiManager &GlowOnnxifiManager::get() {
 }
 
 BackendIdPtr GlowOnnxifiManager::createBackendId(glow::BackendKind kind,
-                                                 bool useOnnx) {
+                                                 bool useOnnx,
+                                                 bool forQuantization) {
   std::lock_guard<std::mutex> lock(m_);
 
-  auto *hostManager = getOrCreateHostManager(kind);
-
-  BackendIdPtr backendId = new onnxifi::BackendId(hostManager, kind, useOnnx);
+  BackendIdPtr backendId;
+  if (forQuantization) {
+    backendId = new onnxifi::InlineBackendId(kind, useOnnx);
+  } else {
+    auto *hostManager = getOrCreateHostManager(kind);
+    backendId = new onnxifi::HostManagerBackendId(hostManager, kind, useOnnx);
```

Comment:
Do we need to care about the cost of virtual function?

Reply:
It's probably not a big deal but if it becomes a problem we can fix it